content/zh/docs/examples/microservices-istio/single/index.md

```diff
@@ -53,24 +41,21 @@ and run it locally:
     $ curl -s {{< github_file >}}/samples/bookinfo/src/ratings/package.json -o package.json
     {{< /text >}}
 
-1. Skim the service's code and note the following elements on the code:
-    - The web server's features:
-        - listening to a port
-        - handling requests and responses
-    - The aspects related to HTTP:
+1. 浏览服务的代码，并注意代码中以下元素：
+    - web 服务器的特征:
+        - 监听一个端口
+        - 处理请求和响应
+    - 与 HTTP 相关的方面:
         - headers
-        - path
-        - status code
+        - 路径
+        - status code 状态码
 
     {{< tip >}}
-    In Node.js, the web server's functionality is embedded in the code of the application. A Node.js
-    web application runs as a standalone process.
+    在 Node.js 中, web服务器的功能嵌入在应用程序的代码中。 一个 Node.js
+    web 应用程序作为一个独立进程运行。
     {{< /tip >}}
 
-1. Node.js applications are written in JavaScript, which means that there is no
-    explicit compilation step. Instead, they use [just-in-time compilation](https://en.wikipedia.org/wiki/Just-in-time_compilation). To build a Node.js application, then means to install its dependencies. Install
-    the dependencies of the `ratings` service in the same folder where you
-    stored the service code and the package file:
+1. Node.js 应用程序是用 JavaScript 编写的， 这意味着没有显式编译步骤。 相反，它们使用 [just-in-time 即时编译](https://en.wikipedia.org/wiki/Just-in-time_compilation)。 要构建 Node.js 应用程序， 则意味着要安装其依赖库。 将 `rating` 服务的依赖库安装在存储服务代码和 package 文件的同一目录下：
```

Comment:
https://en.wikipedia.org/wiki/Just-in-time_compilation
=>
https://zh.wikipedia.org/wiki/%E5%8D%B3%E6%97%B6%E7%BC%96%E8%AF%91